### PR TITLE
Gui: hiding detail dependency information in DlgObjectSelection

### PR DESCRIPTION
**Pull request by realthunder**
Forum thread https://forum.freecadweb.org/viewtopic.php?f=3&t=68023

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6756">pipeline status</a> for feature branch [PR_6756](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6756). Pipeline [517989560](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/517989560) was triggered at [8b295e2](https://github.com/FreeCAD/FreeCAD/pull/6756/commits/8b295e2c899192de5f4eabc41153007999c46c66). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Gui/DlgObjectSelection.ui [hunk 1]
```diff
@@ -158,6 +158,16 @@
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QCheckBox" name="checkBoxShowDeps">
+       <property name="text">
+        <string>Show dependeices</string>
```
Comment (donovaly) [inline src/Gui/DlgObjectSelection.ui:164]: Typo: "dependeices"

**Comment by donovaly** [comment]
Thanks! Despite I like the feature and that it works well for me, it is a new feature and we are in freeze.
-> for 0.21

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6756">pipeline status</a> for feature branch [PR_6756](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6756). Pipeline [518000625](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/518000625) was triggered at [132b06f](https://github.com/FreeCAD/FreeCAD/pull/6756/commits/132b06feac56b18ff55fcc3745fed8581ebecd31). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by realthunder** [comment]
No problem. Just fixed the typo.

**Comment by donovaly** [comment]
OK, merging then.

**Pull request merged**